src/components/form/range/range.tsx

```diff
@@ -203,7 +203,11 @@ export class EuiRangeClass extends Component<
           levels={levels}
           onChange={this.handleOnChange}
           value={value}
-          aria-hidden={showInput === true}
+          aria-hidden={
+            showInput === true || showInput === 'inputWithPopover'
+              ? true
+              : false
+          }
```

Comment:
This could be simplified to 
```suggestion
          aria-hidden={!!showInput}
```
instead, since the only real options are true/false/a string